base_export_manager/models/res_users.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#
+#    Copyright (C) 2015: Ursa Information Systems
+#    Author: Sandip Mangukiya (<smangukiya@ursainfosystems.com>)
+#    Website:(<http://www.ursainfosystems.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+from openerp import api, models
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    @api.v7
```

Comment:
@lasley @max3903 : Sorry Gentlemen, Its was too late last night as per India timings. Well, regarding v8 api, it will call default v7 api itself from web call. But we can add v8 as well as a future support. Am I right ?